src/System.IO.Pipelines/ReadableBuffer.cs

```diff
@@ -40,7 +36,14 @@ public struct ReadableBuffer : ISequence<ReadOnlyMemory<byte>>
         /// <summary>
         /// The first <see cref="Memory{Byte}"/> in the <see cref="ReadableBuffer"/>.
         /// </summary>
-        public Memory<byte> First => _first;
+        public Memory<byte> First
```

Comment:
Don't want I can add a method and deprecate this so it doesn't break immediately.